app/models/user.rb

```diff
@@ -4,4 +4,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :age, numericality: { only_integer: true, greater_than: 17, message: 'You must be older than 18' }
```

Comment:
Given the `greater_than` the message should be "You must be 18 or older" or something along those lines

Reply:
Updated message to "You must be 18 or older"